ReactTask/src/modules/Redux.modules.ts

```diff
@@ -0,0 +1,74 @@
+export interface AuthReducerType {
+  user?: {
+    id: string;
+    first_name: string;
+    last_name: string;
+    photo: string;
+    role_name: string;
+  };
+  token?: string;
+  error?: string;
+  isAuthenticated: boolean;
+}
+
+export interface IAddAppointmentResponse {
+  id: string;
+  patient_id: string;
+  doctor_id: string;
+  visit_date: string;
+  reason: string;
+  note: string;
+  status: string;
+}
```

Comment:
IAddAppointmentResponse  on 14 line and 31?

Reply:
Yea, it duplicate
Removed